docs/config/index.md

```diff
@@ -121,16 +121,32 @@ This might potentially cause some misalignment if a package has different logic
 - **Type:** `boolean`
 - **Default:** `false`
 
-Use [experimental Node loader](https://nodejs.org/api/esm.html#loaders) to resolve imports inside `node_modules`, using Vite resolve algorithm.
+Use [experimental Node loader](https://nodejs.org/api/esm.html#loaders) to resolve imports inside externalized files, using Vite resolve algorithm.
 
-If disabled, your `alias` and `<plugin>.resolveId` won't affect imports inside `node_modules` or `deps.external`.
+If disabled, your `alias` and `<plugin>.resolveId` won't affect imports inside externalized packages (by default, `node_modules`).
 
 #### deps.interopDefault
 
 - **Type:** `boolean`
-- **Default:** `true`
+- **Default:** `false` if `environment` is `node`, `true` otherwise
+
+Interpret CJS module's default as named exports. Some dependencies only bundle CJS modules and don't use named exports that Node.js can statically analyze, when package is imported using `import` syntax instead of `require`. When importing such dependencies in Node environment using named exports, you will see this error:
```

Comment:
```suggestion
Interpret CJS modules default as named exports. Some dependencies only bundle CJS modules and don't use named exports that Node.js can statically analyze when a package is imported using `import` syntax instead of `require`. When importing such dependencies in Node environment using named exports, you will see this error:
```

Reply:
It's talking about `module.exports`, so I'm not sure why it should be plural here?

Reply:
Yeah, that has been reverted.